src/lando/urls.py

```diff
@@ -28,11 +29,16 @@
 urlpatterns += [
     path("__version__", DockerflowViews.VersionView.as_view(), name="version"),
     path("__heartbeat__", DockerflowViews.HeartbeatView.as_view(), name="heartbeat"),
-    path("__lbheartbeat__", DockerflowViews.LoadBalancerHeartbeatView.as_view(), name="lbheartbeat"),
+    path(
+        "__lbheartbeat__",
+        DockerflowViews.LoadBalancerHeartbeatView.as_view(),
+        name="lbheartbeat",
+    ),
```

Comment:
Code formatting problems? How are these still slipping past CI...?

Reply:
See [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1918236) bug.